packages/waku/src/lib/plugins/vite-plugin-rsc-hmr.ts

```diff
@@ -1,22 +1,33 @@
-import path from 'node:path';
-import type { Plugin, ViteDevServer } from 'vite';
+import type { Plugin, TransformResult, ViteDevServer } from 'vite';
 
 const customCode = `
+import { createHotContext as __vite__createHotContext } from "/@vite/client"
+import.meta.hot = __vite__createHotContext(import.meta.url);
+
 if (import.meta.hot && !globalThis.__WAKU_HMR_CONFIGURED__) {
   globalThis.__WAKU_HMR_CONFIGURED__ = true;
+
   import.meta.hot.on('hot-import', (data) => import(/* @vite-ignore */ data));
+
+  import.meta.hot.on('module', (data) => {
+    const code = data.code
+    const script = document.createElement('script')
+    script.type = 'module'
+    script.text = code
+    document.head.appendChild(script)
+  });
```

Comment:
code format inconstant. semi

Reply:
You're correct, I'll fix it after Daishi reviews the code. Thank you.

Reply:
please fix it.